Makefile

```diff
@@ -585,6 +585,17 @@ cpplint:
 
 lint: jslint cpplint
 
+CONFLICT_RE=^>>>>>>> [0-9A-Fa-f]+|^<<<<<<< [A-Za-z]+
+lint-ci: jslint cpplint
+	@if ! ( grep -IEqrs "$(CONFLICT_RE)" benchmark deps doc lib src test tools ) \
```

Comment:
The SmartOS man-page didn't show all flags passed to grep being available, but it seems to work fine anyway.